tests/storages_tests/rdb_tests/test_storage.py

```diff
@@ -251,6 +251,7 @@ def create_test_storage(enable_cache=True, engine_kwargs=None):
 
 def test_pickle_storage():
     # type: () -> None
+    
```

Comment:
[A CI job](https://circleci.com/gh/optuna/optuna/23716?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) failed due to the whitespaces in this line. This CI job is mandatory, and we can not leave this issue even if it is trivial. So, please fix it.